### PR TITLE
Adds hard cider to the bar. 

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
Merge token #18367

Killme

### CODE REVIEW

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -294,6 +294,13 @@
 	icon_state = "lizardwine"
 	list_reagents = list("lizardwine" = 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/hcider
+	name = "Jian Hard Cider"
```
Comment (Niknakflak) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: What's a Jian

Comment (Cheridan) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: it is a mystery to everyone

**Comment by Xhuis** [comment]
That's quite the branch name...

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -658,13 +658,15 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
...
+					/obj/item/weapon/reagent_containers/food/drinks/bottle/absinthe = 5,
 					/obj/item/weapon/reagent_containers/food/drinks/ale = 6,/obj/item/weapon/reagent_containers/food/drinks/bottle/orangejuice = 4,
 					/obj/item/weapon/reagent_containers/food/drinks/bottle/tomatojuice = 4,/obj/item/weapon/reagent_containers/food/drinks/bottle/limejuice = 4,
 					/obj/item/weapon/reagent_containers/food/drinks/bottle/cream = 4,/obj/item/weapon/reagent_containers/food/drinks/soda_cans/tonic = 8,
 					/obj/item/weapon/reagent_containers/food/drinks/soda_cans/cola = 8, /obj/item/weapon/reagent_containers/food/drinks/soda_cans/sodawater = 15,
 					/obj/item/weapon/reagent_containers/food/drinks/drinkingglass = 30,/obj/item/weapon/reagent_containers/food/drinks/ice = 10,
-					/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass = 12,/obj/item/weapon/reagent_containers/food/drinks/flask = 3)
+					/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass = 12,
+					/obj/item/weapon/reagent_containers/food/drinks/flask = 3)
```
Comment (phil235) [inline code/game/machinery/vending.dm]: read line 638 above

Comment (TechnoAlchemisto) [inline code/game/machinery/vending.dm]: My parents never taught me to read, good catch!

**Pull request merged**